common/context.go

```diff
@@ -52,3 +53,14 @@ func GetLaunchOptions(ctx context.Context) *LaunchOptions {
 	}
 	return v.(*LaunchOptions)
 }
+
+// TODO: Test is missing
+func WithProcessID(ctx context.Context, pid int) context.Context {
```

Comment:
Could you document this and `GetProcessID` properly, so that we have less things to do in #58? :)

The `TODO` seems unnecessary as well, we can get to it along with the other missing tests.

Reply:
Sure ;)